Formula/zlint.rb

```diff
@@ -0,0 +1,20 @@
+class Zlint < Formula
+  desc "X.509 certificate linter written in Go"
+  homepage "https://github.com/zmap/zlint"
+  url "https://github.com/zmap/zlint.git",
+      tag:      "v3.4.1",
+      revision: "84da589869bc2d096b7ae42606bf75c602d88806"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "--directory=v3", "GIT_VERSION=v#{version}", "zlint"
+    bin.install "v3/zlint"
+  end
+
+  test do
+    assert_match "ZLint version v#{version}",
+      shell_output("#{bin}/zlint -version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Added a test to have zlint lint a certificate.